internal/server/markdown.go

```diff
@@ -0,0 +1,82 @@
+package server
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"path"
+
+	mermaid "github.com/abhinav/goldmark-mermaid"
+	"github.com/yuin/goldmark"
+	emoji "github.com/yuin/goldmark-emoji"
+	"github.com/yuin/goldmark/extension"
+	"github.com/yuin/goldmark/parser"
+)
+
+var allowedIndexFiles = []string{"README.md", "README.markdown", "readme.md", "readme.markdown", "index.md", "index.markdown"}
```

Comment:
Maybe a map?

Reply:
Same as the situation before re: maps vs slices. Keeping it as-is for the time being.